README.md

```diff
@@ -121,15 +121,35 @@ on how to deploy the project on a live system.
 4. Use the following command if you already have installed the dependencies and want to activate the virtual environment only.
 
 `pipenv shell`
+ 
+5. Use the following commands to create your local database. (If you are using Windows, there will be command variations for the next 3 steps)
 
-5. Run the following command to generate a default user with some pre-existing impacts and locations. This creates an admin user `cfcadmin` with the password `Call4Code`.
+`python3 manage.py makemigrations` 
+
+`python3 manage.py migrate`
```

Comment:
Let's change to `./stage1 migrate` to be in line with the other non-windows commands.

Reply:
done